config/default/editor.editor.basic_html.yml

```diff
@@ -57,11 +57,11 @@ settings:
             - Redo
             - Maximize
   plugins:
+    stylescombo:
+      styles: "a.hs-button|Primary Button\r\na.hs-button--big|Big Primary Button\r\na.hs-secondary-button|Secondary Button\r\na.hs-more-link|More Link\r\na.hs-external-link|External Link\r\np.hs-font-lead|Lead Font\r\np.hs-font-splash|Splash Font\r\np.hs-short-line-length|Short Line Length\r\np.hs-caption|Caption\r\np.hs-credits|Credits\r\nspan.hs-emphasized-text|Emphasized Text\r\ntable.hs-table--borderless|Borderless Table"
```

Comment:
I wonder if it would be helpful to export the new Text Editor configurations for others to use? We could save them in the Google Drive with the Views we've exported.

<img width="1185" alt="export text editor" src="https://user-images.githubusercontent.com/12678977/77969962-111f4680-72b9-11ea-82cb-8391141e64ac.png">

Reply:
so now, this will be a part of all site databases! so no need to add to google drive :)